cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -246,6 +305,33 @@ public void run() {
     }
   }
 
+  private void startTransaction(TransactionContext transactionContext) {
+    try {
```

Comment:
We will not need these methods if we have `runInTx` method for the custom actions.